src/rules.jl

```diff
@@ -150,6 +150,12 @@ end
 @define_diffrule SpecialFunctions.bessely1(x)    =
     :(  (SpecialFunctions.bessely0($x) - SpecialFunctions.bessely(2, $x)) / 2 )
 
+# logerfcx added in SpeciaFunctions 0.10, which requires Julia 1.3
+if VERSION ≥ v"1.3"
```

Comment:
It would be easier, and in my opinion better, to just check if `logerfcx` is defined (this might require an additional `import SpecialFunctions` in src/DiffRules.jl, don't know if it is loaded currently):
```suggestion
# logerfcx was added in SpeciaFunctions 0.10
if isdefined(SpecialFunctions, :logerfcx)
```

Reply:
Do we want to `import SpecialFunctions`? Note that will drag dependencies like OpenSpecFun. What if someone just uses ForwardDiff without needing SpecialFunctions?

But I'll follow your advice here.

Reply:
The package already depends on SpecialFunctions.

Reply:
Yes but it's not imported, is it?

Reply:
Yes, it seems in contrast to LogExpFunctions it is not imported currently. However, since it is a dependency it already pulls in OpenSpecfun etc (BTW ForwardDiff depends on SpecialFunctions directly as well) and eg precompilation already includes SpecialFunctions as well.

Reply:
Ok I can import SpecialFunctions.
However, maybe the issue is that we shouldn't support Julia 1.0? The new LTS is Julia 1.6. If I replace 1.0 in the ci.yml with 1.6 this issue goes away. What do you think?

Reply:
Well nevermind, I'll just do the check you suggest. Can we get this merged as is?

Reply:
IIRC it depends on it only to bound the version, without loading it, this is not an accident. Xref #53

Reply:
Ok what do you suggest then @mcabbott ? I should not load it? How do I check what version of SpecialFunctions is/will be available?

Reply:
Sure, it was not an accident - it wasn't necessary to import it. But now it is.

Reply:
Well, this is one method, but is it necessary? What's the downside to just changing the version bound?

Reply:
IMO the best solution is to load it. There's no need to update the Julia version bound - it only addresses the problem indirectly and we will want to add more rules that were introduced in SpecialFunctions > 1 for which this hack does not work.